vector.lua

```diff
@@ -51,6 +51,35 @@ local function randomDirection(len_min, len_max)
 	                 math.random() * (len_max-len_min) + len_min)
 end
 
+-- < 0 -> counterclockwise
+-- = 0 -> colinear
+-- > 0 -> clockwise
+local function alignment(a, b, c)
```

Comment:
I'll need a bit more thorough a description of what this funtion represents, for example in the docs. From the name alone its purpose is not obvious.